CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
     * Update react-native to version 0.58+ and update App Center SDK packages.
     * Edit **app/build.gradle** to replace `compile` by `implementation`.
     * Run `react-native link` again.
+- **[Feature]** Allow users to set userId that applies to crashes, error and push logs. This feature adds an API, but is not yet supported on the App Center backend.
```

Comment:
nit: `error` -> `errors` to keep the enumeration plural.

Reply:
I'm not sure it's a good idea to keep this statement in the version release notes: `This feature adds an API, but is not yet supported on the App Center backend.`

We're going to support User Id in backend soon, maybe even before some users will see this SDK version. I'm not sure if we can retroactively change the version notes later or what is our plan here?

Reply:
> nit: `error` -> `errors` to keep the enumeration plural.

looks fine grammatically because (logs) is moved out e.g. error **(logs)** and push logs

When it comes to the backend comment, it was @dhei's suggestion to use native change log statement. let's wait for PM's comment.

Reply:
fixed by removing `This feature adds an API, but is not yet supported on the App Center backend.`

Reply:
@nevalenny In general we don't retroactively update CHANGELOG. CHANGELOG only reflects the changes in a specific sdk version.
It really depends on the timeline of sdk release and backend support. If sdk is released before backend support, let's keep the statement to avoid confusion. Otherwise, let's get rid of "This feature adds an API, but is not yet supported on the App Center backend".